packages/sil-governance/src/components/BlockchainTime/index.tsx

```diff
@@ -3,6 +3,8 @@ import { Block, Typography } from "medulas-react-components";
 import React from "react";
 import { ReadonlyDate } from "readonly-date";
 
+export const blockchainTimeHtmlId = "blockchain-time";
```

Comment:
Please use capital letters for this kind of constants. Like BLOCKCHAIN_TIME_HTML_ID

Reply:
@webmaster128 told me to try to move away from that convention.